src/foam/nanos/column/ColumnConfigPropertyValue.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.column',
+  name: 'ColumnConfigToPropertyConverter',
+  documentation: `ColumnConfigToPropertyConverter gathers methods for converting (or mapping) property name to property if required 
+      and returns property and obj, which will return the proprty value on f function call`,
+  javaImports: [
+    'foam.core.ClassInfo',
+    'foam.core.FObject',
+    'foam.core.PropertyInfo',
+    'foam.core.X',
+    'foam.mlang.Expr',
+    'foam.nanos.logger.Logger',
+    'foam.util.StringUtil',
+    'java.lang.reflect.Method'
+  ],
+  methods: [
+    {
+      name: 'filterExportedProps',
+      documentation: 'returns propNames that are properties of the class and aren\'t networkTransient',
+      code: function(of, propNames) {
+        var props = of.getAxiomsByClass(foam.core.Property);
+        var allColumnNames = props.map(p => p.name);
+        if ( ! propNames ) {
+          return props.filter(p => ! p.networkTransient ).map(p => p.name);
+        } props = [];
```

Comment:
This is odd formatting. Why is there a statement on the same line as a }?

Reply:
fixed